tests/integration/rules/symbolicLink.phpt

```diff
@@ -0,0 +1,61 @@
+--FILE--
+<?php
+namespace Respect\Validation\Rules;
```

Comment:
You could've removed that together with the function workaround. Best way is to really have a symbolic link in the repository. It was recently added, btw: `tests/fixtures/symbolic-link`